### DISCUSSION TITLE
Is any way to attach video into gltf/glb?

### DISCUSSION THREAD

**Discussion by challtshan** [google/model-viewer#2804] [Q&A]
Hello,

Recently, I tried to attach video to 3D model with using hotspot with <iframe> inside.
But, it not what i expected, the video itself float on 3D model.

![image](https://user-images.githubusercontent.com/25592401/133756758-d6dfabfe-ea5c-410e-8840-e30569358437.png)

```
<button slot="hotspot-visor" id="hotspot"
   data-position="307.6476457115275m 435.7416834968237m 611.9731713210201m" 
   /*   data-normal="0.29203895750684455 -0.1487151013551118 0.9447735527242781" */>

   <iframe width="200" height="200" src="https://www.youtube.com/somevideo"></iframe>
</button> 
```

I already search for previous discussion, found out that gltf doesn't support video. 

Am looking for possibility so,
Is any way to attach video into gltf/glb 3d model?

**Comment by elalish** [comment]
The result you got is what I would expect 😁. Are you saying you'd like to replace one of  the glTF's textures with a video? We don't support that currently, but I'll consider this a feature request. @timmmeh I wonder how much work this would be to add to our material API?
